Makefile

```diff
@@ -52,6 +52,9 @@ lint-css:
 	cd ${JS_DIR} \
 	&& npm run stylelint
 
+reformat: 
```

Comment:
This specific change is not related to this PR, it seems. It would be more clear to add it as a separate PR.

Reply:
Sound's good. I'll split it then